client/src/App.jsx

```diff
@@ -243,6 +244,7 @@ class App extends React.Component {
             <Route path="/profile" element={<ProfilePopover />} />
             <Route path="*" element={<Error />} />
           </Routes>
+          <Footer />
```

Comment:
It's good so show the footer all the time after login!  Could we have the same conditional rendering like header like `{user? <Footer /> : null `

Reply:
Ok, I changed it. Thank you for noticing that!

Reply:
Thanks Jake